cypress/component/richtext.cy.ts

```diff
@@ -515,6 +568,49 @@ describe('NcRichText', () => {
 			})
 		})
 
+		describe('task lists', () => {
+			it('task list (with `- [ ]` and `- [x]` syntax divided with space from text)', () => {
+				const testCases = [
+					{ input: '- [ ] item 1', output: 'item 1', checked: false },
+					{ input: '- [x] item 2', output: 'item 2', checked: true },
+					{ input: '- [ ] item 3', output: 'item 3', checked: false },
+				]
+
+				mount(NcRichText, {
+					props: {
+						text: testCases.map(i => i.input).join('\n'),
+						useExtendedMarkdown: true,
+					},
+				})
+
+				cy.get('ul').should('exist')
+				cy.get('li').should('have.length', testCases.length)
+				cy.get('li').each((item, index) => {
+					// Vue 2.7 renders three non-breaking spaces here for some reason
+					expect(item).have.text('   ' + testCases[index].output)
```

Comment:
```suggestion
					expect(item).have.text(testCases[index].output)
```
This test fails for me at least locally

Reply:
Probably, should be `contains`

Reply:
Weird, the test seems to run fine on CI, no?

Reply:
Seems to run fine, as we see it. But I believe, Grigorii's proposal should fit here better

Reply:
I also have this issue on master locally.